pkg/store/gcworker/gc_worker.go

```diff
@@ -288,14 +307,24 @@ func (w *GCWorker) tick(ctx context.Context) {
 }
 
 // getGCSafePoint returns the current gc safe point.
-func getGCSafePoint(ctx context.Context, pdClient pd.Client) (uint64, error) {
+func (w *GCWorker) getGCSafePoint(ctx context.Context, pdClient pd.Client) (uint64, error) {
 	// If there is try to set gc safepoint is 0, the interface will not set gc safepoint to 0,
 	// it will return current gc safepoint.
```

Comment:
```suggestion
	// UpdateGCSafePoint returns the current GC safepoint without updating anything if 0 is passed.
```

There's English grammar issue in the comment that was merged years ago.